### PR TITLE
Package ordinal.1.1

### PR DESCRIPTION
**Pull request by RainbowAsteroids**
### `ordinal.1.1`
A language interpreter based on the Forth language
A language interpreter based on the Forth language

---
* Homepage: https://github.com/RainbowAsteroids/ordinal
* Source repo: git+https://github.com/RainbowAsteroids/ordinal.git
* Bug tracker: https://github.com/RainbowAsteroids/ordinal/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ordinal/ordinal.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+description: "A language interpreter based on the Forth language"
+maintainer: ["rainbow@portal2d.xyz"]
+authors: ["Rainbow Asteroids"]
+license: "MIT"
+homepage: "https://github.com/RainbowAsteroids/ordinal"
+bug-reports: "https://github.com/RainbowAsteroids/ordinal/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.3"}
+  "unix"
```
Comment (kit-ty-kate) [inline packages/ordinal/ordinal.1.1/opam:13]: This package does not exist
```suggestion
```

File: packages/ordinal/ordinal.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+opam-version: "2.0"
+synopsis: "A language interpreter based on the Forth language"
+description: "A language interpreter based on the Forth language"
+maintainer: ["rainbow@portal2d.xyz"]
+authors: ["Rainbow Asteroids"]
+license: "MIT"
+homepage: "https://github.com/RainbowAsteroids/ordinal"
+bug-reports: "https://github.com/RainbowAsteroids/ordinal/issues"
+depends: [
+  "ocaml"
```
Comment (kit-ty-kate) [inline packages/ordinal/ordinal.1.1/opam:11]:
```suggestion
  "ocaml" {>= "4.13"}
```
```

#=== ERROR while compiling ordinal.1.1 ========================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.12.1 | pinned(https://github.com/RainbowAsteroids/ordinal/archive/1.1.tar.gz)
# path                 ~/.opam/4.12/.opam-switch/build/ordinal.1.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ordinal -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/ordinal-2060-89a168.env
# output-file          ~/.opam/log/ordinal-2060-89a168.out
### output ###
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.ordinal.objs/byte -no-alias-deps -o lib/.ordinal.objs/byte/ordinal.cmo -c -impl lib/ordinal.ml)
# File "lib/ordinal.ml", line 312, characters 32-48:
# 312 |     | String ss :: state -> if (String.ends_with ~suffix:"\"" word)
#                                       ^^^^^^^^^^^^^^^^
# Error: Unbound value String.ends_with
```

**Comment by kit-ty-kate** [comment]
Thanks

**Comment by kit-ty-kate** [comment]
Could you return the fixes to the opam file upstream so that it can be taken into account for next time?

**Comment by RainbowAsteroids** [comment]
Something like [this](https://github.com/RainbowAsteroids/ordinal/commit/bc01f08c43575423cc585ed867bb4535f071349e)?

**Pull request merged**